src/app/screens/BookDetail/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Text, View, Image, TouchableOpacity } from 'react-native';
+import { RouteProp } from '@react-navigation/native';
+import { RootStackParamList } from '@interfaces/navigation';
+import Routes from '@constants/routes';
+import { BOOKS } from '@constants/mockData';
+import bookPlaceholderCover from '@assets/book_placeholder_cover.png';
+
+import styles from './styles';
+
+interface Props {
+  route: RouteProp<RootStackParamList, Routes.BookDetail>;
+}
+
+function BookDetail({ route }: Props) {
+  const selectedBook = BOOKS.find(({ id }) => id === route.params.bookId);
```

Comment:
Use navigation.getParams to get the navigation parameters

Reply:
Actually, over the weekend I learned this is not the case any more with react navigation 5. Dismiss this comment 😅